pkg/kubeconfig/kubeconfig.go

```diff
@@ -115,6 +115,9 @@ func ForTokenBased(clusterName, clusterID, host, token string) (string, error) {
 }
 
 func ForClusterTokenBased(cluster *managementv3.Cluster, clusterID, host, token string) (string, error) {
+	if cluster.LocalClusterAuthEndpoint == nil {
+		return "", fmt.Errorf("cluster [%s] LocalClusterAuthEndpoint cannot be nil", clusterID)
```

Comment:
Let's update the error to assume nil == false and return `cluster auth endpoint disabled for cluster [%v]`. Saying the field is nil is getting into implementation details that an operator won't understand.